scarb/src/core/manifest/toml_manifest.rs

```diff
@@ -198,6 +198,13 @@ pub struct TomlPackage {
     /// **UNSTABLE** This package does not depend on Cairo's `core`.
     pub no_core: Option<bool>,
     pub cairo_version: Option<MaybeWorkspaceField<VersionReq>>,
+    pub experiments: Option<HashSet<String>>,
```

Comment:
```suggestion
    pub experiments: HashSet<String>,
```

I don't think we'll have to differentiate _none_ from _not specified_, so you can skip the `Option` as well, and use an empty `HashSet` to represent nothing.

FYI empty Vec/HashSet/HashMap etc. do not allocate and are basically free perf wise

Reply:
changed that, all of those are now `Option`s